src/core/packages/chrome/browser-internal/src/ui/project/sidenav_v2/navigation/to_navigation_items.tsx

```diff
@@ -399,15 +412,6 @@ const isRecentlyAccessedDefinition = (
 const filterEmpty = <T,>(arr: Array<T | null | undefined>): T[] =>
   arr.filter((item) => item !== null && item !== undefined) as T[];
 
-const getTestSubj = (navNode: ChromeProjectNavigationNode, isActive = false): string => {
-  const { id, path, deepLink } = navNode;
-  return classnames(`nav-item`, `nav-item-${path}`, {
-    [`nav-item-deepLinkId-${deepLink?.id}`]: !!deepLink,
-    [`nav-item-id-${id}`]: id,
-    [`nav-item-isActive`]: isActive,
```

Comment:
FYI, I'm against using classes for testing. Classnames are more prone to change, so making tests rely on them is a bit of a no-no. What we could assert for the `isActive` case is whether the specific nav item has `aria-current="page"`.